subsystem-test/framework/src/main/java/org/jboss/as/subsystem/test/AbstractSubsystemBaseTest.java

```diff
@@ -159,22 +160,17 @@ public void testSchema() throws Exception {
         SchemaValidator.validateXML(getSubsystemXml(), schemaPath, getResolvedProperties());
     }
 
+    @Test
+    public void testGetSubsystemTemplatePath() throws Exception {
+        Assume.assumeTrue(getSubsystemTemplatePaths().length == 0);
+    }
```

Comment:
`Assume.assumeTrue` is only going to ignore the test if the condition is not satisfied. The Jira description suggests using this as throwaway work to detect which tests are invoking this method returning a non-empty String[].

You don't have to ignore the test if this condition does not occur. Just allow the testsuite to fail. Then create Jiras to clean up WildFly, and then, remove this `testGetSubsystemTemplatePath()` test and ignore both `testSchemaOfSubsystemTemplates()` and `getSubsystemTemplatePaths()`

That's my understanding of the task to accomplish here.

Reply:
Hi @yersan, thanks for clarification, updated.